app/matplotlib.ipynb

```diff
@@ -123,21 +156,36 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
     "categories = [\"Category A\", \"Category B\", \"Category C\", \"Category D\"]\n",
     "sales = [350, 480, 210, 590]\n",
+    "plt.bar(categories, sales)\n",
     "\n",
-    "# write your code here\n"
+    "plt.title(\"Sales for categories\")\n",
+    "plt.xlabel(\"Categories\")\n",
+    "plt.ylabel(\"Sales\")\n",
+    "plt.show()\n"
```

Comment:
In Task 3, you need to rotate the x-ticks by 45 degrees to meet the task requirements. You can do this by adding `plt.xticks(rotation=45)` before `plt.show()`.